scripts/deploy/deployPCVDepositWrapper.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from 'hardhat';
+import { DeployUpgradeFunc } from '@custom-types/types';
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const TOKEN_DEPOSIT = process.env.TOKEN_DEPOSIT;
+  const TOKEN = process.env.TOKEN;
+  const IS_PROTOCOL_FEI_DEPOSIT =
```

Comment:
I don't like adding new env vars that need to be set for deployment scripts that will be re-used; this is fine for now but in the future I want to move to hardhat tasks so that we can pass in these parameters via CLI.